In app update changelog (#1335)
microsoft/appcenter-sdk-dotnet: CHANGELOG.md

# App Center SDK for .NET Change Log

## Version 3.0.1 (Under development)

### App Center Distribute

> * **[Feature]** Add `DisableAutomaticCheckForUpdate` API that needs to be called before SDK start in order to turn off automatic check for update and a `CheckForUpdate` API for a manual update check.

Reviewer: ```suggestion
* **[Feature]** Addded a `DisableAutomaticCheckForUpdate` API that needs to be called before SDK start in order to turn off automatic check for update. 
* **[Feature]** Added a `CheckForUpdate` API to manually check for an update.
```

# App Center SDK for .NET Change Log

> ## Version 3.0.1 (Under development)

Reviewer 2: ```suggestion
## Version 3.1.0 (Under development)
```
